inc/tools.php

```diff
@@ -782,101 +782,101 @@ function cptui_import_types_taxes_settings( $postdata = [] ) {
-						<strong><?php esc_html_e( 'Note:', 'custom-post-type-ui' ); ?></strong> <?php esc_html_e( 'Importing will overwrite previous registered settings.', 'custom-post-type-ui' ); ?>
-					</p>
+	<p>
+		<?php
+			printf(
+				'<strong>%s</strong>: %s',
+				esc_html__( 'NOTE', 'custom-post-type-ui' ),
+				esc_html__( 'This will not export the associated posts or taxonomy terms, just the settings.', 'custom-post-type-ui' )
+			);
+		?>
+	</p>
+	<table class="form-table cptui-table">
+		<?php if ( ! empty( $_GET ) && empty( $_GET['action'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification ?>
+			<tr>
+				<td class="outer">
+					<h2><label for="cptui_post_import"><?php esc_html_e( 'Import Post Types', 'custom-post-type-ui' ); ?></label></h2>
+
+					<form method="post">
+						<textarea class="cptui_post_import" placeholder="<?php esc_attr_e( 'Paste content here.', 'custom-post-type-ui' ); ?>" id="cptui_post_import" name="cptui_post_import"></textarea>
+
+						<p class="wp-ui-highlight">
+							<strong><?php esc_html_e( 'Note:', 'custom-post-type-ui' ); ?></strong> <?php esc_html_e( 'Importing will overwrite previous registered settings.', 'custom-post-type-ui' ); ?>
+						</p>
+
+						<p>
+							<strong><?php esc_html_e( 'To import post types from a different WordPress site, paste the exported content from that site and click the "Import" button.', 'custom-post-type-ui' ); ?></strong>
+						</p>
+
+						<p>
+							<input class="button button-primary" type="submit" value="<?php esc_attr_e( 'Import', 'custom-post-type-ui' ); ?>" />
+						</p>
+						<?php wp_nonce_field( 'cptui_typetaximport_nonce_action', 'cptui_typetaximport_nonce_field' ); ?>
+					</form>
+				</td>
+				<td class="outer">
+					<h2><label for="cptui_post_export"><?php esc_html_e( 'Export Post Types settings', 'custom-post-type-ui' ); ?></label></h2>
+					<?php
+					$cptui_post_types = cptui_get_post_type_data();
+					if ( ! empty( $cptui_post_types ) ) {
+						$content = esc_html( wp_json_encode( $cptui_post_types ) );
+					} else {
+						$content = esc_html__( 'No post types registered yet.', 'custom-post-type-ui' );
+					}
+					?>
+					<textarea title="<?php esc_attr_e( 'To copy the system info, click below then press Ctrl + C (PC) or Cmd + C (Mac).', 'custom-post-type-ui' ); ?>" onclick="this.focus();this.select();" onfocus="this.focus();this.select();" readonly="readonly" aria-readonly="true" class="cptui_post_import" id="cptui_post_export" name="cptui_post_export"><?php echo $content; // phpcs:ignore. ?></textarea>
 
 					<p>
-						<strong><?php esc_html_e( 'To import taxonomies from a different WordPress site, paste the exported content from that site and click the "Import" button.', 'custom-post-type-ui' ); ?></strong>
+						<strong><?php esc_html_e( 'Use the content above to import current post types into a different WordPress site. You can also use this to simply back up your post type settings.', 'custom-post-type-ui' ); ?></strong>
```

Comment:
Not sure what prompted updating the text, outside of perhaps feeling like it needed some clarity.

Reply:
Not sure what happened here. I didn't change the text.

Reply:
I'm not sure what went wrong here. But I think, they are different lines for a different purposes. And while fixing, their line number changed!